tests/Microsoft.Data.Analysis.Tests/DataFrameTests.cs

```diff
@@ -1654,30 +1606,30 @@ public void TestMerge()
             merge = left.Merge<int>(right, "Int", "Int");
             Assert.Equal(merge.Rows.Count, left.Rows.Count);
             Assert.Equal(merge.Columns.Count, left.Columns.Count + right.Columns.Count);
-            Assert.Equal(merge["Int_right"][6], right["Int"][6]);
+            Assert.Equal( merge.Columns["Int_right"][6], right.Columns["Int"][6]);
             VerifyMerge(merge, left, right, JoinAlgorithm.Left);
 
             // Right merge 
             merge = left.Merge<int>(right, "Int", "Int", joinAlgorithm: JoinAlgorithm.Right);
             Assert.Equal(merge.Rows.Count, right.Rows.Count);
             Assert.Equal(merge.Columns.Count, left.Columns.Count + right.Columns.Count);
-            Assert.Equal(merge["Int_right"][2], right["Int"][2]);
-            Assert.Null(merge["Int_left"][12]);
+            Assert.Equal( merge.Columns["Int_right"][2], right.Columns["Int"][2]);
+            Assert.Null( merge.Columns["Int_left"][12]);
             VerifyMerge(merge, left, right, JoinAlgorithm.Right);
 
             // Outer merge 
             merge = left.Merge<int>(right, "Int", "Int", joinAlgorithm: JoinAlgorithm.FullOuter);
             Assert.Equal(16, merge.Rows.Count);
             Assert.Equal(merge.Columns.Count, left.Columns.Count + right.Columns.Count);
-            Assert.Null(merge["Int_left"][12]);
-            Assert.Null(merge["Int_left"][5]);
+            Assert.Null( merge.Columns["Int_left"][12]);
+            Assert.Null( merge.Columns["Int_left"][5]);
             VerifyMerge(merge, left, right, JoinAlgorithm.FullOuter);
 
             // Inner merge 
             merge = left.Merge<int>(right, "Int", "Int", joinAlgorithm: JoinAlgorithm.Inner);
             Assert.Equal(9, merge.Rows.Count);
             Assert.Equal(merge.Columns.Count, left.Columns.Count + right.Columns.Count);
-            Assert.Equal(merge["Int_right"][2], right["Int"][2]);
+            Assert.Equal( merge.Columns["Int_right"][2], right.Columns["Int"][2]);
```

Comment:
Are there unnecessary whitespace changes here?

Reply:
Yup. Search and replace artifact. Good catch!